source/code/include/playfab/PlayFabPlatformMacros.h

```diff
@@ -9,6 +9,10 @@
 
 #pragma once
 
+// TODO: is there a replacement that we can test instead?
+#ifdef _GAMING_DESKTOP
```

Comment:
Reflecting on this, I suspect this is only defined on Windows GDK builds, not Console. @jasonsandlin do you know if there's a good token to determine the presence of the GDK on Console?

Reply:
libHttpClient does this:
    #if defined(_GAMING_DESKTOP) || defined(_GAMING_XBOX) || defined(_GAMING_XBOX_XBOXONE) || defined(_GAMING_XBOX_SCARLETT)
if any of those 3 are defined, then we infer its a GDK build.  

for this project, if you want PC GDK builds only use 
#if defined(_GAMING_DESKTOP)

If you want console GDK builds only  then use
#if defined(_GAMING_XBOX) || defined(_GAMING_XBOX_XBOXONE) || defined(_GAMING_XBOX_SCARLETT)

if you want PC & console GDK builds, then use all 4 or'd

Reply:
We want to support both with this, so we'll need all four.